### DISCUSSION TITLE
Version for the University of Minho School of Engineering

### DISCUSSION THREAD

**Discussion by zohannn** [joaomlourenco/novathesis#153] [Q&A] [answered]
Hello everyone.
Is there a release for the University of Minho/ School of Engineering ?
Thank you.
Best Regards,

**Comment by joaomlourenco** [comment, marked as answer]
Hi,

There is a fork from @b-pereira for “uminho”, but this fork is based in version 4.x, which does not compile anymore in tex-live-2020.

The template is currently in version 5.x, which was a major upgrade (to support tex-live-2020 and many other features), but does not support **yet** “uminho”.

I'm planning to add support for “uminho” in v5.x in the very near future (until the end of Feb 2021).

If you are starting your thesis/dissertation, it is 100% safe to use the template as is (without support for uminho, just select any other school) and later upgrade when the support is available.  

If you are closing/ending your thesis/dissertation, I advise you to use the existing fork from @b-pereira.

**Reply by RicardoAlexandreMartins** [reply]
Hi !

I'm interested about this support for "uminho". Do you have any news about it ?

**Reply by joaomlourenco** [reply]
Yes!  😊

Yesterday I started working on it…  but found a limitation of the new version that I have to overcome.  

Just to make it clear, "uminho" specifications require parametrizing memoir for "single side (oneside)", also font size 12pt insted of 11pt… 

So I went back to the main version to find a way to be able to configure memoir from the "defaults.ldf" file in each school's folder.  And that is a considerable challenge because I've to postpone loading the "memoir" class for a while, and this requires revising the order by which commands are defained and files are loaded.  In the end it became slightly harder to do than initially planed.  But it is almost done…  :)

Once the above is ok, I'll get back to “uminho” version.  I hope to be done in a couple of weeks.

**Reply by joaomlourenco** [reply]
A small report on the progress!
Up to now:
- fonts are ok (works with XeLaTeX and LuaLaTeX)
- cover for PhD and MSc is done
- the back of the cover page is always done (empty page with the right color)
- the second cover (_folha de rosto_) is done
- the back covers (equal to the froont covers but with no text) are also done

See image below…
![cover-examples](https://user-images.githubusercontent.com/2064643/113492478-74951300-94cf-11eb-800f-aa4a1fb9916e.jpg)

**Reply by b-pereira** [reply]
@joaomlourenco 
1.  The second cover (_folha de rosto_) must be, all of it, in the *PANTONE  Cool Gray 7 C*;
2.  The second cover (_folha de rosto_) logos must be a part of each other 0.5mm;
3. The PhD cover fonts must be in white colour;

### Examples:
#### Example of item 1 and item 2
![front page](https://user-images.githubusercontent.com/944511/113518594-37885980-957f-11eb-9c09-ba9d74162cb0.png)
#### Example of item 3
![PhD cover](https://user-images.githubusercontent.com/944511/113518617-5850af00-957f-11eb-944a-e9e0f6ce30a3.png)

**Comment by joaomlourenco** [comment]
Support for UMinho is available in the master version of he repository.
Just edit "Config/1_novathesis" and uncomment "\ntsetup{school=uminho/ee}".
To make a MSc dissertation, uncomment "\ntsetup{docdegree=msc}" in the same file.

Please report any problem you may by creating an new issue.  Thanks.
